SEMOSS/semoss-ui
# 591 fix remove extra layer on variables in construct and ensure variables can be referenced in

How to test:

Step 1.
While on the dev branch, I created old apps based on the apps that were built on workshop dev.  I grabbed the json from there and pulled them into my local.  Did this for about 10 apps.

Step 2. 
Switch to the branch tied to pull request and ensure the old state hits my migration function as intended.

Resolves #18
